source/js/components/search-form.js

```diff
@@ -0,0 +1,101 @@
+import fetchUtils from "@degjs/fetch-utils";
+
+const searchForm = () => {
+
+
+    const formEl = document.querySelector('.js-search-form');
+    const layoutEl = document.querySelector('.js-general-content')
+
+    const url = formEl.dataset.api;
+
+    function init() {
+        formEl.addEventListener('submit', e => {
+            e.preventDefault();
+            getData();
+        });
+    }
+
+
+    function getData() {
+        const formEls = formEl.querySelectorAll('.js-value');
+        fetchUtils.getJSON(
+            url,
+            {
+                body: JSON.stringify(formEls),
+                method: 'POST',
+                headers: {
+                    'Content-Type': 'application/json'
+                }
+            },
+            {
+                timeout: 10000
```

Comment:
[minorfix] I think this is the plugin default

Reply:
Removed!